File: Python/2023/6. June/beecrowd 1038.py
a = int(input())
if a == 1:
    x = 4
    
elif a == 2:
    x = 4.50
    
elif a == 3:
    x = 5
    
elif a == 4 :
    x = 2
    
elif a == 5 :
    x = 1.50
    
b = int(input())
c = float(x*b) 
d = ("%.2f" % c)

print(f"Total: R$ {d}")

File: C++/2024/3. March/Beecrowd/beecrowd 1038.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){

    float menu[5] = {4.00, 4.50, 5.00, 2.00, 1.50};
    
    int a; cin >> a;
    int b; cin >> b;


    cout << "Total: R$ " << fixed << setprecision(2) << menu[a-1]*b << endl;


    return 0;
}